<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColordKCM</class>
 <widget class="QWidget" name="ColordKCM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>344</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="4">
        <widget class="QToolButton" name="removeProfileBt">
         <property name="text">
          <string>Remove Profile</string>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="profilesTb">
         <property name="text">
          <string>Profiles</string>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="6">
        <widget class="QStackedWidget" name="tabWidget">
         <widget class="QWidget" name="page_1">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTreeView" name="devicesTV">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="animated">
              <bool>true</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTreeView" name="profilesTV">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="expandsOnDoubleClick">
              <bool>false</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="addProfileBt">
         <property name="text">
          <string>Add Profile</string>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="devicesTb">
         <property name="text">
          <string>Devices</string>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="info_page">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>249</width>
            <height>146</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>249</width>
            <height>69</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="KTitleWidget" name="infoWidget">
          <property name="text">
           <string notr="true">You have no devices created</string>
          </property>
          <property name="comment">
           <string notr="true">This is probably a problem</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="profile_page">
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="Description" name="profile" native="true"/>
        </item>
        <item row="0" column="0">
         <spacer name="offsetSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
  <customwidget>
   <class>Description</class>
   <extends>QWidget</extends>
   <header location="global">Description.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>devicesTV</tabstop>
  <tabstop>devicesTb</tabstop>
  <tabstop>profilesTb</tabstop>
  <tabstop>removeProfileBt</tabstop>
  <tabstop>profilesTV</tabstop>
  <tabstop>addProfileBt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
